### ISSUE TITLE
Changing an existing theme file for FreshRSS

### ISSUE THREAD

**Issue by jfmessierdotca** [FreshRSS/FreshRSS#2090]
FreshRSS is a great product, and I am using one of the newly added theme in my FreshRSS installation, and I really like it. However, one thing I am looking for is a smaller line heights, so have a maximum of article headers displayed on a page without scrolling. I never took any formal training or even did reading on CSS. All my files are on a remote VPS, which runs my FreshRSS install, and so my basic access is this good old terminal and vi. 

On one of the newly added theme, I see a .CSS and a .SCSS file, The second seems to be the "source code" of the first one. How can I edit the first one to have thinner lines for the article titles in the normal view ? Which tags should I look for ? Also, should I change the .SCSS file and "compile" it, or should I edit the .CSS file directly ? 

Thank You :-)

**Comment by Alkarex** [comment]
Hello @jfmessierdotca 
For minor, custom changes, I suggest you add a few custom rules in our "CustomCSS" extension:
https://github.com/FreshRSS/Extensions/tree/master/xExtension-CustomCSS
(I am actually considering including this extension along with the FreshRSS core, as it is a quite usual one people want to install)

**Comment by jfmessierdotca** [comment]
Very interesting....... I definitely think I would want to go this way. However, I realize I will have a lot of reading to do to understand CSS in general and then understand how those custom extensions work in the context of FreshRSS. Lately, I was doing changes directly in the .CSS file, and testing to see whether the effects were the ones expected. Absolutely not the way to go, but was giving me the results I wanted. However, I think it's more than time to actually do those changes the right way.  
  
  
Thanks :-)

**Comment by Frenzie** [comment]
If you right click somewhere → Inspect element you can try out changes more conveniently. ;-)

**Comment by jfmessierdotca** [comment]
This is what I did to find out the file and location where everything was defined. I have to say that Firefox and Chrome work a little different on this, but I managed to get the changes i wanted by changing the base template file as opposed to the actual theme CSS files. I now have the display I want............... I guess this is something that could be there as an easy-to-choose user option. In this case, it was the templace.css file from the base-theme directory under ./FreshRSS/theme directory, line 505.  
  
Thanks :-)
